src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -18,6 +17,7 @@ import 'holofuel/global-styles/index.css'
 
 function PrimaryLayout ({
   children,
+  color,
```

Comment:
doesn't look like you're using this prop?

Reply:
This was residual for a non-final design decision... good catch.